### PR TITLE
fix: Add CSP headers to dashboard to prevent click jacking

### PR DESCRIPTION
**Pull request by perkinsjr**
## What does this PR do?

Fixes # (issue)

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [Contributing Guide](./CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand areas
- [ ] Ran `pnpm build`
- [ ] Ran `pnpm fmt`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

### CODE REVIEW

File: apps/dashboard/next.config.js [hunk 1]
```diff
@@ -19,6 +26,14 @@ const nextConfig = {
     // your project has ESLint errors.
     ignoreDuringBuilds: true,
   },
+  async headers() {
```
Comment (chronark) [inline apps/dashboard/next.config.js:29]: does this include `/` or not? I can never remember

**Comment by chronark** [comment]
We should add this to the app too, no?

**Comment by perkinsjr** [comment]
We don't have a `/` on the dashboard

**Comment by perkinsjr** [comment]
But it includes everything

**Comment by chronark** [comment]
ah sorry, this is dashboard
what about web?

**Comment by chronark** [comment]
www*

**Comment by perkinsjr** [comment]
We can do web if we need too, but it doesn't contain any click jackable entries e.g auth

**Comment by chronark** [comment]
ok, I have no idea about this
🤷

**Comment by chronark** [comment]
I’ll greenlight

**Review by chronark — APPROVED** [review]

**Pull request merged**